### DISCUSSION TITLE
Not receiving the launch token while creating new account using organization email id as the primary email

### DISCUSSION THREAD

**Discussion by vjadhav13** [community/community#76224] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello, I am trying to create a new account as per our organization process, but I am not receiving the token on my company email id I am providing. I tried multiple times, also if I try to create new account again and provide the same email id, it says email id already taken.

Can you please help me here on priority as I am stuck in creating the account using my company email id for further steps, as we have moved now from bitbucket to github and I we are supposed to use github.

**Comment by vjadhav13** [comment]
Any help here please, not receiving the launch code in the organization email id provided as the primary email while creating the github account.

**Comment by Miguel-Sanz** [comment]
Hello vjadhav13,

I understand the challenges you are facing with creating a new account and not receiving the token on your company email for the GitHub migration. Let's troubleshoot this together to get you up and running as soon as possible.

Here are a few steps you can take to resolve this issue:

Check Spam/Junk Folder:
Ensure that the token email hasn't been redirected to your Spam or Junk folder. Sometimes, automated emails can be filtered.

Correct Email:
Confirm that the email address you provided during the account creation matches your company email exactly. Typos can lead to delivery issues.

Resend Token:
If you've checked the spam folder and verified the email, try using the platform's option to resend the token. Look for a "Resend Token" or similar link during the account creation process.

Contact Support:
If the issue persists, reach out to GitHub support directly. They can provide personalized assistance and help you troubleshoot any platform-specific problems.

Remember, it's crucial to use your company email as it might be linked to organizational processes. If the system says the email is already taken, it's possible that an account was partially created or there is a glitch.

If none of these steps resolve your issue, I recommend reaching out to GitHub support at [GitHub Support Contact] or checking their official documentation for troubleshooting account creation problems.
